### PR TITLE
chore(governance): align post-cutover constitutional surfaces

### PR DESCRIPTION
**Pull request by jamesryancooper**
## What

Align the accepted post-cutover constitutional remediation across the live
constitutional families, support/disclosure claims, bootstrap authority
surfaces, subordinate governance ownership, and fail-closed validator paths.

## Why

The accepted proposal required one atomic remediation pass that removed stale
phase-era live selectors, narrowed live claims to retained proof, and made
recurrence fail closed.

No-Issue: accepted proposal execution requested directly by operator.

## How

Rebound active family manifests to the March 30 atomic selector while
preserving earlier phase receipts as explicit lineage, corrected bootstrap
wording so raw `inputs/**` never appear as authored authority, narrowed support
and HarnessCard claims to the proved repo-local consequential envelope, and
added new fail-closed validators wired into both the alignment path and the
assurance gate.

## Profile Selection Receipt

- Semantic version source(s): `version.txt`, `.octon/framework/constitution/charter.yml`
- Release state (`pre-1.0` or `stable`): `pre-1.0`
- `change_profile` (`atomic` or `transitional`): `atomic`
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance): no downtime; single-branch atomic merge unit; no staged coexistence layer; rollback is revertable as one commit; blast radius is `.octon/**` constitutional/governance/assurance surfaces only
- Tie-break status: no tie-break required
- `transitional_exception_note` (required when `pre-1.0` + `transitional`): `n/a`

## Implementation Plan

- Workstreams and scope:
  - normalize active constitutional family live selectors
  - preserve disclosure-root semantics and narrow support/live claim width
  - remove bootstrap authored-authority leakage into raw inputs
  - normalize subordinate governance owners to durable identifiers
  - add and wire fail-closed validators plus retained publication receipts
- Public interfaces/types/contracts affected:
  - constitutional family manifests
  - support-target declaration and HarnessCard disclosure surfaces
  - alignment/assurance runtime validators and closure assertion surfaces
- Test scenarios:
  - stale family selector drift
  - bootstrap/raw-input authority leakage
  - live support overclaim and disclosure parity drift
  - placeholder governance owner drift
  - alignment and assurance gate execution
- Assumptions/defaults:
  - the March 30 atomic cutover remains the only live selector
  - truthful narrowing beats preserving aspirational live claims

## Impact Map (code, tests, docs, contracts)

- Code/tests:
  - added new fail-closed validators under `.octon/framework/assurance/runtime/_ops/scripts/`
  - updated existing validation scripts and closure assertion logic
- Docs:
  - `.octon/README.md`
  - `.octon/framework/constitution/CHARTER.md`
  - `.octon/instance/bootstrap/START.md`
  - `.octon/instance/charter/workspace.md`
  - subordinate governance docs under `.octon/framework/cognition/governance/**`
- Contracts:
  - active family manifests under `.octon/framework/constitution/contracts/**`
  - `.octon/instance/governance/support-targets.yml`
  - `.octon/instance/governance/closure/unified-execution-constitution.yml`
  - authored and retained HarnessCard surfaces
- Retained evidence:
  - `.octon/state/evidence/validation/publication/unified-execution-constitution-closure/**`
  - `.octon/state/evidence/runs/retention.json`

## Compliance Receipt

- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (`atomic`) unless hard gates required `transitional`.
- [x] Included `transitional_exception_note` when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations

none

## Tradeoffs

This PR deliberately narrows previously broader live support framing to the
retained proved envelope. The only remaining warning in local validation is the
preexisting empty retained run directories reported by `validate-continuity-memory.sh`.

## Testing

- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-constitutional-family-live-model.sh` -> PASS
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-bootstrap-authority-surfaces.sh` -> PASS
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-support-target-live-claims.sh` -> PASS
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-disclosure-live-roots.sh` -> PASS
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-subordinate-owner-identifiers.sh` -> PASS
- `bash .octon/framework/assurance/governance/_ops/scripts/assert-unified-execution-closure.sh` -> PASS
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-assurance-disclosure-expansion.sh` -> PASS
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-objective-binding-cutover.sh` -> PASS
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-runtime-lifecycle-normalization.sh` -> PASS
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-continuity-memory.sh` -> PASS with warning-only preexisting empty retained run dirs
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-runtime-effective-state.sh` -> PASS
- `bash .octon/framework/assurance/runtime/_ops/scripts/alignment-check.sh --profile harness` -> PASS (`errors=0`)
- `bash .octon/framework/assurance/runtime/_ops/scripts/compute-assurance-score.sh --weights .octon/framework/assurance/governance/weights/weights.yml --scores .octon/framework/assurance/governance/scores/scores.yml --charter .octon/framework/assurance/governance/CHARTER.md --context .octon/framework/assurance/governance/weights/inputs/context.yml --profile ci-reliability --run-mode ci --maturity beta --repo octon --out-dir .octon/generated/.tmp/assurance/engine-alignment` -> PASS
- `bash .octon/framework/assurance/runtime/_ops/scripts/assurance-gate.sh --scorecard .octon/generated/.tmp/assurance/engine-alignment/scorecard.yml --weights .octon/framework/assurance/governance/weights/weights.yml --scores .octon/framework/assurance/governance/scores/scores.yml --charter .octon/framework/assurance/governance/CHARTER.md --baseline-weights .octon/framework/assurance/governance/weights/weights.yml --baseline-scores .octon/framework/assurance/governance/scores/scores.yml --baseline-charter .octon/framework/assurance/governance/CHARTER.md --mode ci --summary-out .octon/generated/.tmp/assurance/engine-alignment/gate-summary.md` -> PASS

## Rollout

`n/a` — atomic `.octon/**` governance and validation alignment on a feature
branch.

## Convivial Purpose Check

- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric

- Risk class: [ ] Trivial [ ] Low [x] Medium [ ] High
- Rollback plan: revert commit `bce1baab3`
- Rollback handle: `git revert bce1baab3`
- Flags changed (name, owner, expiry, rollout): none
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model

- OpenAPI/JSON-Schema changes: no API schema changes; constitutional/support/disclosure contract semantics updated
- Threat model update/link: no separate threat-model artifact added; fail-closed enforcement and support-boundary narrowing are part of this remediation

## Observability and Performance

- Traces/logs/metrics for changed flows: validation and assurance gate output only
- Representative traces for high risk changes: retained closure publication receipts and assurance gate summary
- Performance or bundle impact: negligible; additional validator execution only

## License and Provenance

- New dependencies and licenses: none
- Generated code/templates provenance notes: retained publication receipt added under existing closure publication surface

## Checklist

- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [x] All review conversations resolved

## Screenshots / Notes

No visual changes.

### CODE REVIEW

File: .octon/framework/assurance/runtime/_ops/scripts/validate-support-target-live-claims.sh [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  require_yq '.host_adapters[] | select(.adapter_id == "studio-control-plane" and .support_status == "experimental" and .default_route == "stage_only")' "$SUPPORT_TARGETS" "studio host is demoted to experimental"
+  require_yq '.host_adapters[] | select(.adapter_id == "github-control-plane" and .support_status == "experimental" and .default_route == "stage_only")' "$SUPPORT_TARGETS" "GitHub host is demoted to experimental"
+  require_yq '.host_adapters[] | select(.adapter_id == "ci-control-plane" and .support_status == "experimental" and .default_route == "stage_only")' "$SUPPORT_TARGETS" "CI host is demoted to experimental"
+  require_yq '.model_adapters[] | select(.adapter_id == "repo-local-governed" and .support_status == "supported" and .default_route == "allow")' "$SUPPORT_TARGETS" "repo-local-governed remains the supported model adapter"
+  require_yq '.model_adapters[] | select(.adapter_id == "experimental-external" and .support_status == "experimental" and .default_route == "stage_only")' "$SUPPORT_TARGETS" "experimental external model remains staged"
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/assurance/runtime/_ops/scripts/validate-support-target-live-claims.sh:185]: Implemented. `validate-support-target-live-claims.sh` now proves exclusivity as well as presence: it requires exactly one supported host adapter and exactly one supported model adapter, and asserts those unique supported adapters are `repo-shell` and `repo-local-governed` respectively. Revalidated locally before pushing `7fd70806c`.

File: .octon/framework/assurance/runtime/_ops/scripts/validate-constitutional-family-live-model.sh [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  local family_id="$2"
+  local receipt="$3"
+
+  if rg -Fq -- "$receipt" "$family_file"; then
+    if yq -e ".activation_lineage_refs[] | select(. == \"$receipt\")" "$family_file" >/dev/null 2>&1; then
+      pass "${family_id} preserves historical lineage for ${receipt##*/migrations/}"
+    else
+      fail "${family_id} mentions ${receipt##*/migrations/} outside activation_lineage_refs"
+    fi
+  fi
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/assurance/runtime/_ops/scripts/validate-constitutional-family-live-model.sh]: Implemented. `validate-constitutional-family-live-model.sh` no longer checks lineage opportunistically from string presence. It now requires the expected historical receipt explicitly per active family (`objective`/`authority` -> phase 2, `runtime`/`retention` -> phase 3, `assurance` -> phase 4, `adapters` -> phase 5), so removing a required lineage receipt fails closed. Revalidated locally before pushing `7fd70806c`.
